### PR TITLE
Add Trust: Mildaurion impl

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

https://www.bg-wiki.com/ffxi/Cipher:_Mildaurion
https://ffxiclopedia.fandom.com/wiki/Trust:_Mildaurion

Went to retail to get skill IDs and observe.

At naked 99, these were the HP/MP:
image

Pretty much in line with what I got once I adjusted her job. (the +100 MPP works)

Tested friendship lines (or whatever they're called), they work as intended.

Put her next to Naji to see if her skills were ridiculous, since there are already Lua files for them since they're ROTZ, but they didn't seem broken.

Her skill-based text chat works.

She'll get her traits from her jobs.

### CODE REVIEW

File: scripts/enum/mob_skills.lua [hunk 1]
```diff
@@ -67,6 +67,8 @@ xi.mobSkill =
 
     SHEEP_SONG_3      = 3433,
 
+    LIGHT_BLADE       = 3471,
```
Comment (Xaver-DaRed) [inline scripts/enum/mob_skills.lua:70]: :heart:

**Review by Xaver-DaRed — DISMISSED** [review]
A minor nitpick, we have `:isTrust()` but its the same thing really, just shorter

File: scripts/actions/mobskills/great_wheel.lua [hunk 2]
```diff
@@ -20,7 +20,11 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.SLASHING, xi.mobskills.shadowBehavior.NUMSHADOWS_3)
 
     target:takeDamage(dmg, mob, xi.attackType.PHYSICAL, xi.damageType.SLASHING)
-    mob:resetEnmity(target)
+
+    if not target:getObjType() == xi.objType.TRUST then
```
Comment (Xaver-DaRed) [inline scripts/actions/mobskills/great_wheel.lua]: oh, wait, no this cant be done
`not` is exclusive for boolean returns. cant use with comparators

Comment (claywar) [reply scripts/actions/mobskills/great_wheel.lua]: Technically it can be, but it leads to confusion (and why CI calls us out on it).  But easiest is ~=

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**